angular/pubspec.yaml

```diff
@@ -30,25 +30,18 @@ dependencies:
   stack_trace: ^1.6.5
   tuple: ^1.0.1
 
+# TODO(alorenzen): Move lib/src/testing/matchers.dart to _tests and remove this.
```

Comment:
This isn't the angular_compiler pubspec, it's the angular pubspec.

Also, this is just a PR to push changes to github, everything has already been reviewed internally.